modules/config/config.scm

```diff
@@ -94,4 +94,18 @@ end-of-c-declare
  (gambit-c (if (string=? (system-platform) "android") (##heartbeat-interval-set! -1.)))
  (else (if (string=? (system-platform) "android") (##set-heartbeat-interval! -1.))))
 
+(cond-expand
+ (android
+  (c-declare #<<EOF
+extern char* android_getFilesDir_info_get();
+char* android_getFilesDir_info()
```

Comment:
You are now making a third way of calling `android_getFilesDir()` - am I missing something that makes this one different? as otherwise only the `extern char* android_getPackageCodePath()` needs to stay here?